src/modules/main/sidebar/portal/Portal.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { NavLink } from 'react-router-dom';
+
+import { Button } from '@material-ui/core';
+import Cancel from '@material-ui/icons/Cancel';
+
+import csx from './Portal.scss'
+import csx2 from '../Sidebar.scss'
+
+const LINK_HEIGHT = 80;
+
+const MARKER_HEIGHT = 30;
+
+export const Portal = ({sidebarLinks, basePath, close}) => {
+    return (
+        <div className={csx.portal}>
+
+            <figure className={csx2.logo}>
+                <Cancel fontSize='large' onClick={close}/>
+            </figure>
+
+            <div className={csx2.links}>
+
+            {sidebarLinks.map(({ path, label, icon, exact }) => (
+                <NavLink
+                key={label}
+                exact={exact}
+                activeClassName={csx2.active}
```

Comment:
You should create a component - SiedbarLInk which contains repeated parts - f.e styles active, link and this component allows to put children dynamically - to render content inside

Reply:
Component has beed created